docs/src/main/paradox/cassandra.md

```diff
@@ -206,11 +206,12 @@ The database schema for the offset storage table:
 
 ```
 CREATE TABLE IF NOT EXISTS akka_projection.offset_store (
-  projection_id text,
+  projection_name text,
+  projection_key text,
   offset text,
   manifest text,
   last_updated timestamp,
-  PRIMARY KEY (projection_id))
+  PRIMARY KEY (projection_name, projection_key))
```

Comment:
This means that all projections with the same `projection_name` will end up in the same Cassandra partition. Maybe not a big problem, but maybe we should consider adding a `partition` column. 
```
PRIMARY KEY ((projection_name, partition), projection_key)
```

partition can be an Int, let's say that we have 5 fixed partitions (1,2,3,4,5). It can be assigned from
```
math.abs(projectionId.id.hashCode % 5) + 1
```

It makes the read all that is needed for MergableOffset more difficult, but if we keep it to a low fixed number that shouldn't be a problem. It would have to select 5 queries with (projection_name, n).

wdyt @chbatey ?

Reply:
Is the projection key the partitioning of the projection e.g. the tag for events by tag?

I guess this won't be read often but may be written to every event so will become a hotspot if one partition. 

Would making adding the projection key to the partition key be potentially too many?

Reply:
We could add the projection id back (name + key) in addition to separate fields for name and key.

Reply:
The normal query is (name,key), but we will probably have to also support reading all by (name). The latter doesn’t work if we add the key to the partioning key.

That’s why I was thinking of spreading it out to 5.

Reply:
Or could we just swap the ordering of the name and key?  I don't think the read use needs to be optimized at all for this table.

Reply:
You cant query without the partition key, i.e. you csn’t query by name if the id is the partition key.

Partition key is the first field in the primary key. Or several if within parens, ((proj_name, partition), proj_key)

Reply:
Ok, got it.

Reply:
I implemented the partition field.